src/components/typography/text/text.styles.js

```diff
@@ -48,15 +48,15 @@ const getTone = tone => {
 const getElementFontSize = elementType => {
   switch (elementType) {
     case 'h1':
-      return '32px';
+      return '2.461538461rem';
     case 'h2':
-      return '24px';
+      return '1.8461538461rem';
     case 'h3':
-      return '20px';
+      return '1.5384615384rem';
     case 'h4':
-      return '16px';
+      return '1.2307692307rem';
     case 'h5':
-      return '14px';
+      return '1.0769230769rem';
```

Comment:
Yeah, I mostly put them like this so Percy would pass. We could just as easily set them now to max 3 digits and then just approve the failing tests?

Reply:
That why I thought of adding them as calc. It kind of gives a better insight into the ratios. Can obv we a follow up or we round.

Reply:
As long as we let the design team know about it and that they're fine with it.

We can also do that as a follow up, I don't mind.

Reply:
I think it's sort of weird to do the calc because that's sort of what REM is, calculated font sizes. Feels weird to send a calc to the calc so it can calc? 🤔 

![yo dawg](https://i.imgflip.com/2s835q.jpg)

Reply:
Ok, I smiled you funny, lovely... coffee nerd.